### PR TITLE
Add path to grab specific podcasts from stations

### PR DESCRIPTION
**Pull request by PrinceOm**

### CODE REVIEW

File: api/station/index.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ const controller = require('./station.controller');
 const router = express.Router();
 
 router.get('/', controller.list);
+router.get('/:stationId', controller.find);
```
Comment (dting) [inline api/station/index.js]: lets change this route to:
```
router.get('/:stationId/podcasts/', controller.listPodcasts);
```

File: api/station/station.controller.js [hunk 2]
```diff
@@ -9,4 +10,10 @@ controller.list = (req, res) => {
     .catch(handleError(res));
 };
 
+controller.find = (req, res) => {
+	Podcast.find({station: req.params.stationId}).exec()
+		.then(podcasts => res.json(podcasts))
+		.catch(handleError(res))
+}
```
Comment (dting) [inline api/station/station.controller.js]: Missing semi colon. Run linter.

File: api/station/station.controller.js [hunk 3]
```diff
@@ -9,4 +10,10 @@ controller.list = (req, res) => {
     .catch(handleError(res));
 };
 
+controller.find = (req, res) => {
```
Comment (dting) [inline api/station/station.controller.js]: rename this to `listPodcasts`.

**Comment by dting** [comment]
LGTM.

File: api/station/station.controller.js (hunk 2 above)
Comment (PrinceOm) [inline api/station/station.controller.js]: Done.

File: api/station/index.js (hunk 1 above)
Comment (PrinceOm) [inline api/station/index.js]: Done.

File: api/station/station.controller.js (hunk 3 above)
Comment (PrinceOm) [inline api/station/station.controller.js]: Done.

**Pull request merged**